core/src/identity.rs

```diff
@@ -349,9 +333,9 @@ mod tests {
     #[test]
     fn keypair_from_protobuf_encoding() {
         // E.g. retrieved from an IPFS config file.
-        let base_64_encoded = "CAESQL6vdKQuznQosTrW7FWI9At+XX7EBf0BnZLhb6w+N+XSQSdfInl6c7U4NuxXJlhKcRBlBw9d0tj2dfBIVf6mcPA=";
+        let base_64_encoded = "RAgBEkC+r3SkLs50KLE61uxViPQLfl1+xAX9AZ2S4W+sPjfl0kEnXyJ5enO1ODbsVyZYSnEQZQcPXdLY9nXwSFX+pnDw";
         let expected_peer_id =
-            PeerId::from_str("12D3KooWEChVMMMzV8acJ53mJHrw1pQ27UAGkCxWXLJutbeUMvVu").unwrap();
```

Comment:
Why does this test change? We should not need to change this test.

Reply:
This changes because of this https://github.com/libp2p/rust-libp2p/pull/3066#issuecomment-1340132206. I've been thinking about this and feel uneasy about it. Should we instead use this pattern https://github.com/tafia/quick-protobuf/issues/202#issuecomment-1093701486? That way we're consistent with the official encoding.

Reply:
> Should we instead use this pattern [tafia/quick-protobuf#202 (comment)](https://github.com/tafia/quick-protobuf/issues/202#issuecomment-1093701486)? That way we're consistent with the official encoding.

I think so yes. We need to be compatible here so this test should not need changing!